### PR TITLE
perf(i18next): switch to new backend library

### PR DESCRIPTION
**Pull request by favna**
Goodbye library written in the year 0, hello library written in 2022.

### CODE REVIEW

**Review by vladfrangu — DISMISSED** [review]

**Comment by favna** [comment]
@sapphiredev pack

**Review by vladfrangu — DISMISSED** [review]

**Review by kyranet — DISMISSED** [review]

**Comment by Nytelife26** [comment]
I'm gonna request a hold on this until I can review it later today if that's okay. Thanks in advance

File: package.json [hunk 1]
```diff
@@ -27,29 +27,28 @@
...
-		"jest": "^27.5.1",
-		"jest-circus": "^27.5.1",
+		"jest": "^28.1.0",
 		"lint-staged": "^12.4.1",
 		"prettier": "^2.6.2",
 		"pretty-quick": "^3.1.3",
 		"rimraf": "^3.0.2",
-		"ts-jest": "^27.1.4",
+		"ts-jest": "^28.0.1",
 		"ts-node": "^10.7.0",
```
Comment (imranbarbhuiya) [inline package.json:50]: Ig you can remove ts-node now.

**Review by imranbarbhuiya — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (favna) [reply package.json:50]: good catch

**Comment by imranbarbhuiya** [comment]
I missed you can remove the resolutions too.

**Pull request merged**